Update Labels: Issue Template: wiki--research-plan-review

kiwookim opened the pull request:
Fixes #4488

### What changes did you make?
  - replaced labels with: ['Feature: Wiki', 'role: user research', 'size: 0.5pt', 'Complexity: Missing'] in this file path .github/ISSUE_TEMPLATE/wiki--research-plan-review----replace-with-name-of-page-.md
  

### Why did you make the changes (we will use this info to test)?
  - To avoid GitHub bot taking off the labels when dev adds them to issues
  

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
 - No Visual Changes to the website






For PR Reviewers and Merge Team
To review this issue, click the link below under "Link for reviewers" and verify that the labels match the updated labels string in the issue.

Link for Reviewers
This URL will be inserted by the dev during the test process and will be used by reviewers to preview the updated template.
URL of the issue branch on the test Repository: https://github.com/kiwookim/website/issues/new?assignees=sacamp&labels=Feature%3A+Wiki%2Crole%3A+user+research%2Csize%3A+0.5pt%2CComplexity%3A+Missing&projects=&template=wiki--research-plan-review----replace-with-name-of-page-.md&title=Wiki%3A+Research+Plan+Review%3A++%5BREPLACE+WITH+NAME+OF+PAGE%5D

For PM, Merge Team, or Tech Lead

 Once the pull request associated with this issue is approved and merged, please update and edit epic https://github.com/hackforla/website/issues/4307 by
 Checking off the dependency for this issue
 If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label
File and Code links you will need to work on this issue
Directory to find the page in once you have it in your IDE: .github/ISSUE_TEMPLATE/wiki--research-plan-review----replace-with-name-of-page-.md
This issue is part of epic https://github.com/hackforla/website/issues/4307

kiwookim committed:
added labels to blank issue form with dependency yml file

kiwookim committed:
replaced labels .github/ISSUE_TEMPLATE/wiki--research....

adrianang commented:
- Availability: MTThFSS, 6 PM - 10 PM (PST) (Week of 7/10)
- ETA: I plan on finishing this PR review by Tuesday, 7/11 at 10 PM.

adrianang reviewed (changes requested):
Hi @kiwookim! The branching is set up correctly for this PR, and the corresponding issue is also linked, but in reviewing the files changed, I noticed that you also edited `.github/ISSUE_TEMPLATE/blank-issue-form-with-dependency.yml`. This change seems to correspond with issue #4485 instead, so changes on that file should be discarded for this issue; only `.github/ISSUE_TEMPLATE/wiki--research-plan-review----replace-with-name-of-page-.md` should be changed.

Other than that, the requested code has been changed correctly and the template linked to in the reviewer URL is coming up with the correct labels pre-selected. There are some other changes in the file, but they seem to be cosmetic as they are just removing trailing spaces and will ultimately help clean up the file! Once the changes to the blank issue form are discarded, this should be good to go for merging.

blulady commented:
eta: Thursday
avaiability: Thursday

kiwookim commented:
> Hi @kiwookim! The branching is set up correctly for this PR, and the corresponding issue is also linked, but in reviewing the files changed, I noticed that you also edited `.github/ISSUE_TEMPLATE/blank-issue-form-with-dependency.yml`. This change seems to correspond with issue #4485 instead, so changes on that file should be discarded for this issue; only `.github/ISSUE_TEMPLATE/wiki--research-plan-review----replace-with-name-of-page-.md` should be changed.
> 
> Other than that, the requested code has been changed correctly and the template linked to in the reviewer URL is coming up with the correct labels pre-selected. There are some other changes in the file, but they seem to be cosmetic as they are just removing trailing spaces and will ultimately help clean up the file! Once the changes to the blank issue form are discarded, this should be good to go for merging.

Hi @adrianang, thank you for the review! Thank you for pointing that out. I actually was wondering about that. Before making a new branch, I git pulled from gh-pages, then made a new branch and only locally changed the file associated with this issue.  But, as you mentioned, I saw all these other changes when I made PR. Cosmetic changes might be from the extension Prettier formatter I am using, but the other labels were the result of 'git pull.' To revert back the file(.github/ISSUE_TEMPLATE/blank-issue-form-with-dependency.yml), should I just delete the labels it added when I git pulled and push again?  Also, what would be a good workflow to prevent this for future PR? Thanks a lot!

mademarc commented:
Review ETA: 7/13/2023
Availability: 7:20PM

mademarc reviewed (dismissed):
Hey @kiwookim the changes for line 3 on the yml was done correctly, the same goes for the line 5 changes for the .md file plus the format was properly done well.

adrianang commented:
@kiwookim I took a look at the commit history for your branch and noticed that all of the changes in the blank template GHA file came from commit `fab9469`, so I think the easiest fix to do would be to run `git revert fab9469` in your branch and push that commit into this branch again!

`git revert [commit ID]` makes a new commit that undoes the changes made in the [commit ID] commit  only, which works well in this case since we can discard all the changes in `fab9469` (blank issue template) and want to keep the changes in `55298d2` (research plan template).

I think a good workflow to prevent this for future PRs would be to check what files are being changed when you're making a new PR on Github and double-checking that you've only made changes to files expected in the linked issue. Individually adding files with `git add` would be good for prevention, and knowing how to `git revert` certain commits would be good to 'fix' commits that have already been made 😃

kiwookim committed:
Revert "added labels to blank issue form with dependency yml file"

This reverts commit fab946942c8d34570808096d1a26858194664cc7.

:wq

kiwookim commented:
@adrianang Thanks a lot for the detailed explanation. It makes a lot of sense now! I will be careful only to add the files I changed in future issues. @mademarc @adrianang I reverted the changes and pushed again! Thank you, adrian and marco!

adrianang reviewed (approved):
This looks good! The branching is still setup correctly, the correct issue is linked, and the changes to the blank template were discarded, so only the Wiki Research Plan template is being changed now. The labels requested to be added are showing up when I check the template through the link. Thank you @kiwookim for taking up this issue and quickly making the requested change! 🙌🏽

t-will-gillis reviewed (approved):
Hi @kiwookim - Great job! The branches are good as is your description, and you have changed only what is being asked for in the issue (besides the removal of whitespaces which you said is probably Prettier). Great communication between yourself and @adrianang to get your commits correct! 
Thanks!
